NER_Data_collection_and_annotation.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<p>why not add <code>gsutil</code> in <code>requirements.txt</code> to skip line 9-10?</p><p> </p><p>Ideally, this should be a bash script, but this is fine.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meetnaren/ner-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>When I created the notebook I didn't have the repo in mind. Going forward I think the dataset will probably become part of the repo, so we could potentially eliminate this step altogether.</p>